backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -1080,8 +1081,10 @@ class SubmissionDatabaseService(
             .fetchSize(streamBatchSize)
             .asSequence()
             .map {
-                val metadata = it[originalMetadata]
-                val selectedMetadata = fields?.associateWith { field -> metadata[field] }
+                // Revoked sequences have no original metdadata, hence null can happen
+                @Suppress("USELESS_ELVIS")
```

Comment:
oh wow that is weirdest suppress argument, I thought it was a mistake initially!